tests/php/sync/test_class.jetpack-sync-full.php

```diff
@@ -117,6 +114,27 @@ function test_full_sync_sends_all_comments() {
 		$this->assertEquals( 11, count( $comments ) );
 	}
 
+	function test_full_sync_expands_comment_meta() {
+		$post_id = $this->factory->post->create();
+		list( $comment_ID ) = $this->factory->comment->create_post_comments( $post_id );
+
+		add_comment_meta( $comment_ID, 'hc_post_as', 'wordpress', true );
+		add_comment_meta( $comment_ID, 'hc_wpcom_id_sig', 'abcd1234', true );
+		add_comment_meta( $comment_ID, 'hc_foreign_user_id', 55, true );
+
+		$this->server_replica_storage->reset();
+		$this->client->reset_data();
+
+		$this->full_sync->start();
+		$this->client->do_sync();
+
+		$comment = $this->server_replica_storage->get_comments()[0];
```

Comment:
I am pretty sure that PHP 5.2 will complain about `get_comments()[0];`
I think we need to rewrite 
```
$comments = $this->server_replica_storage->get_comments()[0];
$comment = $comments[0];
```